torch_lr_finder/lr_finder.py

```diff
@@ -185,7 +189,11 @@ def range_test(
         iter_wrapper = DataLoaderIterWrapper(train_loader)
         for iteration in tqdm(range(num_iter)):
             # Train on batch and retrieve loss
-            loss = self._train_batch(iter_wrapper, accumulation_steps)
+            loss = self._train_batch(
+                iter_wrapper,
+                accumulation_steps,
+                non_blocking_transfer=non_blocking_transfer,
+            )
             if val_loader:
                 loss = self._validate(val_loader)
```

Comment:
Missing `non_blocking_transfer=non_blocking_transfer,`

Reply:
Good catch! Fixed it and squashed the new commit.